### ISSUE TITLE
[Record page] Availability table should have an aria-label=availability

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#3905]
### Steps to recreate
reviewed this with @sandbergja 
Open the screenreader rotor in the record page. [Read](https://support.apple.com/guide/voiceover/navigate-webpage-tables-vo27958/mac) the table. The user should know that it's an availability table.

### URL
https://catalog.princeton.edu/catalog/994264203506421

### Issues
The user doesn't know what is contained in the table until they listen to it.

### WCAG Conformance level (A, AA, AAA)
[Understanding SC 4.1.2](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)

### User impact
The user doesn't know what is contained in the table until they listen to it.

### Recommended fix
Add aria-label='availability'

### Acceptance criteria
- [x] I can successfully use this feature with Voice Over in Safari.
- [x] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...

**Comment by christinach** [comment]
related to #3643
